tests/test_basic.py

```diff
@@ -261,6 +276,27 @@ def cli(flag):
     assert result.output == expect
 
 
+@pytest.mark.parametrize(
+    ("default", "args", "expect"),
+    (
+        # See: https://github.com/pallets/click/issues/3024#issuecomment-3146199461
```

Comment:
Describe what's being tested, not only a link.

Reply:
Done in: https://github.com/pallets/click/pull/3030/commits/c498ced0e7b595880b70a81767ae86902714ee1c